content/docs/1.6.0/advanced-resources/rwx-workloads.md

```diff
@@ -31,22 +31,55 @@ It is necessary to meet the following requirements in order to use RWX volumes.
 
-# Notice
-
-In versions 1.4.0 to 1.4.3 and 1.5.0 to 1.5.1 of Longhorn, Longhorn CSI plugin `hard` mounts a Longhorn volume exported by a NFS server located within a share-manager Pod in the `NodeStageVolume`. The `hard` mount allows NFS requests to persistently retry without termination, ensuring that IOs do not fail. When the server is back online or a replacement server is recreated, the IOs resume seamlessly, thus guaranteeing data integrity. However, there is potential risk that to maintain file system stability, the Linux kernel will not permit unmounting a file system until all pending IOs are written back to storage, and the system cannot undergo shutdown until all file systems are unmounted. If the NFS server fails to recover successfully, the client nodes must undergo a forced reboot.
-
-To address this stability problem, Longhorn switches to adopt the use of a `softerr` mount with a `timeo` value of 600 and a `retrans` value of 5 as default options since v1.4.4, 1.5.2 and 1.6.0. When the NFS server becomes unreachable due to factors such as node power outages, network partitions and so on, NFS clients will fail an NFS request after the specified number of retransmissions, resulting in an NFS `ETIMEDOUT` error being returned to the calling application and potential data loss. If `softerr` is not supported, Longhorn will automatically revert to using the `soft` option instead.
-
-Please refer to [#6655](https://github.com/longhorn/longhorn/issues/6655) for more information.
-
 # Creation and Usage of a RWX Volume
 
+An RWX volume must have its access mode set to `ReadWriteMany` and not have the migratable flag set.
+
 1. For dynamically provisioned Longhorn volumes, the access mode is based on the PVC's access mode.
 2. For manually created Longhorn volumes (restore, DR volume) the access mode can be specified during creation in the Longhorn UI.
 3. When creating a PV/PVC for a Longhorn volume via the UI, the access mode of the PV/PVC will be based on the volume's access mode.
 4. One can change the Longhorn volume's access mode via the UI as long as the volume is not bound to a PVC.
 5. For a Longhorn volume that gets used by a RWX PVC, the volume access mode will be changed to RWX.
 
+## Setting the volume mount options
+
+The volume must be mounted via NFS to be accessed, and by default Longhorn mounts with NFS version 4.1, with options `softerr`, a `timeo` value of 600, and a `retrans` value of 5.  If the NFS server becomes unreachable due to factors such as node power outages, network partitions and so on, NFS clients will fail an NFS request after the specified number of retransmissions, resulting in an NFS `ETIMEDOUT` error being returned to the calling application and potential data loss. If `softerr` is not supported, Longhorn will automatically revert to using the `soft` option instead, which will return an `EIO` in error conditions.
```

Comment:
Should we proactively change this to 4.2 before merging?

Reply:
Yes, I'm going to wait and see which releases v4.2 is backported to.

Reply:
Done.